renderer/pages/settings.js

```diff
@@ -178,36 +178,51 @@ class Settings extends React.Component {
             <Container pt={3}>
               <Heading h={4}><FormattedMessage id='Settings.Sharing.Label' /></Heading>
-              <BooleanOption
-                onConfigSet={this.reloadConfig}
-                label={<FormattedMessage id='Settings.Sharing.UploadResults' />}
-                optionKey='sharing.upload_results'
-                config={config} />
-
-              <BooleanOption
-                onConfigSet={this.reloadConfig}
-                label={<FormattedMessage id='Settings.Sharing.IncludeNetwork' />}
-                optionKey='sharing.include_asn'
-                config={config} />
-
-              <BooleanOption
-                onConfigSet={this.reloadConfig}
-                label={<FormattedMessage id='Settings.Sharing.IncludeCountryCode' />}
-                optionKey='sharing.include_country'
-                config={config} />
-
-              <BooleanOption
-                onConfigSet={this.reloadConfig}
-                label={<FormattedMessage id='Settings.Sharing.IncludeIP' />}
-                optionKey='sharing.include_ip'
-                config={config} />
-
-              <NumberOption
-                onConfigSet={this.reloadConfig}
-                label={<Text>Websites tested (0 means all)</Text>}
-                optionKey='nettests.websites_url_limit'
-                config={config} />
-              <Text pt={3}>OONI Probe Desktop v{pkgJson.version}</Text>
+              <Flex flexDirection='column'>
+
+                <BooleanOption
+                  onConfigSet={this.reloadConfig}
+                  label={<FormattedMessage id='Settings.Sharing.UploadResults' />}
+                  optionKey='sharing.upload_results'
+                  config={config}
+                />
+
+                <BooleanOption
+                  onConfigSet={this.reloadConfig}
+                  label={<FormattedMessage id='Settings.Sharing.IncludeNetwork' />}
+                  optionKey='sharing.include_asn'
+                  config={config}
+                />
+
+                <BooleanOption
+                  onConfigSet={this.reloadConfig}
+                  label={<FormattedMessage id='Settings.Sharing.IncludeCountryCode' />}
+                  optionKey='sharing.include_country'
+                  config={config}
+                />
+
+                <BooleanOption
+                  onConfigSet={this.reloadConfig}
+                  label={<FormattedMessage id='Settings.Sharing.IncludeIP' />}
+                  optionKey='sharing.include_ip'
+                  config={config} />
+
+                <NumberOption
+                  onConfigSet={this.reloadConfig}
+                  label={<Text>Websites tested (0 means all)</Text>}
+                  optionKey='nettests.websites_url_limit'
+                  config={config}
+                />
+
+                <BooleanOption
+                  onConfigSet={this.reloadConfig}
+                  label={<Text>Collect analytics data using Matomo</Text>}
```

Comment:
This string is missing from the translations. Can we use a string which we already have in the mobile app which is already translated?

Reply:
@hellais We don't have a string for this in the mobile app either. 

Perhaps we can replace the copy of this string with:

Collect app usage statistics

Reply:
added